### ISSUE TITLE
[BUG] The draft folder is being indexed as a separate version

### ISSUE THREAD

**Issue by soee** [TYPO3-Documentation/t3docs-search-indexer#70]
After the latest changes, we now have the `draft` folder indexed as a separate version.
This happens for example for the `georgringer/news` extensions.
The `draft` folder should be excluded from indexing.

**Comment by linawolf** [comment]
Yes, and it used to be excluded, wondering how it made its way back in

**Comment by linawolf** [comment]
Resolved
